kafka/pack.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: kafka
+description: Integration pack for Apache Kafka message broker
+keywords:
+  - kafka
+  - queuing
+  - messaging
+  - message broker
+version: 0.1.0
```

Comment:
Can you bump this version up to 0.2.0? So it doesn't get confused with https://github.com/StackStorm/st2-kafka

You could also add yourself as a contributor? (Or not, up to you)